README.md

```diff
@@ -3,6 +3,7 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 ## Available Scripts
+This is setup
 
 In the project directory, you can run:
```

Comment:
- [ ] **Required:** Kindly [update your readme file.](https://github.com/microverseinc/readme-template) This makes your project look more professional.